webpack.common.js

```diff
@@ -34,6 +35,18 @@ module.exports = {
 					'sass-loader'
 				]
 			},
+			{
+				test: /\.(css|scss)$/,
+				use: [
+					MiniCssExtractPlugin.loader,
+					'css-loader',
+					'postcss-loader',
+					'sass-loader'
+				]
+			},
```

Comment:
Hrmm when I made this I was sure I tested it thoroughly by importing the code into the project where I want to use it... but now (using example.html), the CSS can only get included in the bundle if I don't include this extra rule...
Is there a good way I don't know of to sort of "in parallel" extract the css to a separate file and also include it in the bundle?